nursery/get-password-database-entry-on-linux.yml

```diff
@@ -4,6 +4,7 @@ rule:
     namespace: host-interaction/session
     authors:
       - michael.hunhoff@mandiant.com
+      - jonathanlepore@google.com
     scopes:
       static: basic block
```

Comment:
let's make this `instruction` and same with the other rules, since they each only look for a single call instruction.